lgc/state/PipelineState.cpp

```diff
@@ -263,6 +263,24 @@ void PipelineState::mergePalMetadataFromBlob(llvm::StringRef blob, bool isGlueCo
     m_palMetadata->mergeFromBlob(blob, isGlueCode);
 }
 
+// =====================================================================================================================
+// Set the "other part-pipeline" from the given other Pipeline object. This is used when doing a part-pipeline
+// compile of the non-FS part of the pipeline, to get the FS input mappings from the FS part-pipeline.
```

Comment:
If it really does something this specific shouldn't the function name be closer to that.  That way, when people see this function called, they have a better idea of what it is going to do.  A name like `getFsInputMappingFromPipeline` would be more informative.

Reply:
Maybe, but I didn't want to tie the concept of "here's the other part-pipeline elf" to exactly what we're going to extract from it, in case we want to extract anything else in the future. I don't have any specific plans for doing that.

Reply:
If the function is meant to be that general, then why would the comments mentions the specifics?  I think that either a name change or a comment change would be good.

Reply:
I removed the over-specific comments.